contracts/contracts/strategies/ICurvePool.sol

```diff
@@ -15,6 +15,8 @@ interface ICurvePool {
         uint256 _minAmount
     ) external;
 
+    function remove_liquidity(uint256 _amount, uint256 _minAmount) external;
```

Comment:
Actual ABI is `def remove_liquidity(_amount: uint256, min_amounts: uint256[N_COINS])`, an array.

Reply:
Thanks dvf 👍